server/fishtest/static/js/application.js

```diff
@@ -195,6 +195,24 @@ async function fetch_text(url, options) {
   return response.text();
 }
 
+async function process_title(count) {
+  // async because the document title is often set in javascript
+  await DOM_loaded();
+  title = document.title;
```

Comment:
```js
let title = document.title;
```

Reply:
All fixed. BTW it might been better to set the title in the html header
```
<title>
Title here ...
</title>
```
Then this function `process_title` wouldn't have to be async.